lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -2974,6 +2974,16 @@ $(document).ready(function() {
 	$('.no-highlight').folderListing({
 		enableHighlight: false,
 	});
+
+	$('a[href*="access/basiclti"]').each(function() {
+		var href = this.href;
```

Comment:
One more small suggestion:
```suggestion
		const href = this.href;
```